engine/kv_engine.cpp

```diff
@@ -380,9 +383,19 @@ Status KVEngine::RestoreSortedRecord(DLDataEntry *pmem_data_entry,
         return Status::MemoryOverflow;
       }
       new_hash_offset = (uint64_t)dram_node;
+      if (configs_.opt_restore_sorted &&
+          thread_res_[write_thread.id]
+                      .visited_skiplist_ids_[dram_node->GetSkipListId()]++ %
+                  10000 ==
```

Comment:
Can add some explanation of the hard code 10000?

Reply:
I added